modules/cluster-logging-rn-5.6.4.adoc

```diff
@@ -0,0 +1,32 @@
+//module included in cluster-logging-release-notes.adoc
+:_content-type: REFERENCE
+[id="cluster-logging-release-notes-5-6-4_{context}"]
+= Logging 5.6.4
+This release includes link:https://access.redhat.com/errata/RHBA-2023:1311[OpenShift Logging Bug Fix Release 5.6.4].
+
+[id="openshift-logging-5-6-4-bug-fixes"]
+== Bug fixes
+* Before this update, when LokiStack was deployed as the log store, the logs generated by Loki pods were collected and sent to LokiStack. With this update, the logs generated by Loki are excluded from collection and will not be stored. (link:https://issues.redhat.com/browse/LOG-3280[LOG-3280])
+
+* Before this update, the static LokiStack memberlist configuration relied solely on private IP networks. As a result, when the {product-title} cluster pod network was configured with a non-private IP network range, the LokiStack pods would crashloop. With this update, the LokiStack administrator now has the option to use the pod network for the memberlist configuration. This resolves the issue and prevents the LokiStack pods from crashlooping when the {product-title} cluster pod network is configured with a non-private IP network range. (link:https://issues.redhat.com/browse/LOG-3454[LOG-3454])
+
+* Before this update, when the `tls.insecureSkipVerify` option was set to `true`, the Cluster Logging Operator would generate incorrect configuration. As a result, the operator would fail to send data to Elasticsearch when attempting to skip certificate validation. With this update, the Cluster Logging Operator generates the correct TLS configuration even when `tls.insecureSkipVerify` is enabled. As a result, data can be sent successfully to Elasticsearch even when attempting to skip certificate validation. (link:https://issues.redhat.com/browse/LOG-3475[LOG-3475])
+
```

Comment:
I think the second sentence in the write-up for LOG-3640 reads more clearly if we drop the word "only"

This resulted in some of the received logs including the structured message, while others did not.